alpha-build-machinery/make/examples/multiple-binaries/manifests/operator.openshift.io_myoperatorresources.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
I could go for simpler one but why not

Reply:
this was just the kubebuilder init example